File: src/main/java/com/graqr/threshr/model/redsky/stores/Hour.java
package com.graqr.threshr.model.redsky.stores;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.core.annotation.Nullable;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record Hour(
        @Nullable @JsonProperty("begin_date")
        String beginDate,
        @JsonProperty("begin_time") String beginTime,
        @JsonProperty("end_date") String endDate,
        @JsonProperty("end_time") String endTime) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/store/Hour.kt
package com.graqr.threshr.circleStore.model.store

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.core.annotation.Nullable
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class Hour(
    @param:Nullable @param:JsonProperty("begin_date")
    val beginDate: String,
    @param:JsonProperty("begin_time") val beginTime: String,
    @param:JsonProperty("end_date") val endDate: String,
    @param:JsonProperty("end_time") val endTime: String
) {
}
